astroquery/vizier/tests/test_vizier_remote.py

```diff
@@ -38,9 +38,16 @@ def test_query_region(self):
     def test_query_region_async(self):
         response = vizier.core.Vizier.query_region_async(
             self.target, radius=5 * u.deg, catalog=["HIP", "NOMAD", "UCAC"])
-
         assert response is not None
-
+        response = vizier.core.Vizier.query_region_async(
```

Comment:
It's best to make a new function for this, like so (I hope I got the whitespace right; if you like this, you can just merge it):
```suggestion

    
    def test_query_region_async_galactic(self):
        response = vizier.core.Vizier.query_region_async(
```